app/code/local/Excellence/Test/Model/Test3.php

```diff
@@ -0,0 +1,10 @@
+<?php
+class Excellence_Test_Model_Test extends Mage_Core_Model_Abstract
```

Comment:
@vibhor1111 right now there are many files name Test1, Test2, etc etc

remove this. make a model and table named "users" 
with columns firstname, lastname, email, password, dob